camcal:
  calibration_setup:
    calsheet: ARUCOCALPICT_A
    xEgo_offset_mm: 1010

  ego_target:
    egorange_x_mm: [500, 2000]
    egorange_y_mm: [-1500, 1500]
    egoframewidth_px: 1200
  ipm:
    H_rowmajor: '[[2.0241071055221362, 10.311415293642327, -878.0559308544815], [-0.2621300866245443,
      13.080056903779683, -1136.7010764719914], [-0.00044540266608232104, 0.017035640265587685,
      1.0]]'
    ego_width_px: 1200
    ego_height_px: 600
    xiEgo0_px: 600.0
    etaEgo0_px: 800.0
    ego_scale_mm2px: 0.4
  roi:
    ego_roi_px: '[[0, 0], [7.273202380587236, 0.0], [1200, 0], [1200.0, 29.919302112192447],
      [740.531243034392, 600.0], [474.5882167990943, 600.0]]'
    img_roi_px: '[[-8.087337903849045, 86.74130015128901], [8.041264854433899e-14,
      86.9033739557757], [1255.970047830746, 112.07356548151525], [1280.0, 118.14213327235902],
      [1280.000000000001, 605.2313414234266], [-8.064300111409441e-14, 607.520073732699]]'
